R/snapshot.R

```diff
@@ -462,7 +462,7 @@ renv_snapshot_validate_dependencies_available <- function(project, lockfile, lib
 
   })
 
-  renv_pretty_print(
+  notice(
```

Comment:
Most of the feedback in this file appears to be reporting a problem with the active environment; should they be `caution`?